tests/collections/collection.test.ts

```diff
@@ -725,8 +917,48 @@ describe(`StargateMongoose - ${testClientName} Connection - collections.collecti
       assert.ok(findOneRespDoc!._id);
       assert.ok(ids.has(findOneRespDoc!._id));
         });
-        it('should find & find doc $exists test', async () => {
-      interface Doc { _id?: string; username: string; city: string; tags?: string[] }
+        it('should find & find doc $nin test', async () => {
+      interface Doc {
```

Comment:
Very strange indentation here, can you please fix?

Reply:
fixed